### ISSUE TITLE
Error when running radiation-daysim with reference case

### ISSUE THREAD

**Issue by emanuel030** [architecture-building-systems/CityEnergyAnalyst#1816]
I encounter an error when running `cea radiation-daysim` from the terminal: 

```
Daysim simulation starts for building(s) [u'B01', u'B02', u'B03', u'B04', u'B05', u'B06', u'B07', u'B08', u'B09']
and the next number of total sensors 2349
=======================================================
= gen_dc: calculate direct daylight coefficients      =
= (this simulation may take several minutes to hours) =
=======================================================
calculate 65 direct daylight coefficients for variant static_system...
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
=======================================================
= gen_dc: calculate diffuse daylight coefficients     =
= (ignore WARNING: no light sources found)            =
= (this simulation may take several minutes to hours) =
=======================================================
calculate diffuse daylight coefficients for variant static_system...
The system cannot find the path specified.
The system cannot find the path specified.
The system cannot find the path specified.
ds_illum: fatal error: File c:\users\emanuel\appdata\local\temp\temp0\res\temp0.dc does not contain any uncommented lines.

IN CASE YOU ARE USING DAYSIM UNDER WINDOWS, THE "RAPYPATH" VARIALBE MIGHT NOT HAVE BEEN SET CORRECTLY. TO TEST THIS, OPEN A DOS PROMPT AND TYPE 'set'. AMONG THE VARIALBLES DISPLAYED SHOULD BE 'Raypath c:\Daysim\lib'.
PLEASE REFER TO THE DAYSIM TUTORIAL FOR MORE INFORMATION...Traceback (most recent call last):
  File "C:\Users\emanuel\Miniconda2\envs\cea\Scripts\cea-script.py", line 11, in <module>
    load_entry_point('cityenergyanalyst', 'console_scripts', 'cea')()
  File "c:\users\emanuel\documents\github\cityenergyanalyst\cea\interfaces\cli\cli.py", line 65, in main
    script_module.main(config)
  File "c:\users\emanuel\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\radiation_main.py", line 213, in main
    radiation_singleprocessing(rad, geometry_3D_zone, locator, config.weather, settings)
  File "c:\users\emanuel\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\radiation_main.py", line 166, in radiation_singleprocessing
    daysim_main.isolation_daysim(chunk_n, rad, building_dict, locator, weather_path, settings)
  File "c:\users\emanuel\documents\github\cityenergyanalyst\cea\resources\radiation_daysim\daysim_main.py", line 157, in isolation_daysim
    solar_res = rad.eval_ill_per_sensor()
  File "C:\Users\emanuel\Miniconda2\envs\cea\lib\site-packages\py4design\py2radiance\__init__.py", line 1147, in eval_ill_per_sensor
    res_dict_list = self.eval_ill()
  File "C:\Users\emanuel\Miniconda2\envs\cea\lib\site-packages\py4design\py2radiance\__init__.py", line 1119, in eval_ill
    ill_file = open(ill_path, "r")
IOError: [Errno 2] No such file or directory: 'c:\\users\\emanuel\\appdata\\local\\temp\\temp0\\res\\temp0.ill'
```

The temp0.dc looks like this: 
```
# merged daylight coefficients from c:\users\emanuel\appdata\local\temp\temp0\tmp\/static_system.dif.dc and c:\users\emanuel\appdata\local\temp\temp0\tmp\/static_system.dir.dc.
# latitude: 47.18
# direct:	 # c:/daysim\bin\rtrace_dc  -h -I+ -oc -aa 0.150000 -ab 4 -ad 512 -ar 20 -as 32 -dj 0.700000 -dr 0 -ds 0.000000  -lr 8 -lw 0.050000  -sj 1.000000 -st 0.500000    -L 1000.000000  -N 65 -Dm -dt 0.000000  "c:\users\emanuel\appdata\local\temp\temp0\tmp\/static_system.dir.oct" < "c:\users\emanuel\appdata\local\temp\temp0\pts\points_0.pts" >> "c:\users\emanuel\appdata\local\temp\temp0\tmp\/static_system.dir.dc"
# diffuse:	 # c:/daysim\bin\rtrace_dc  -h -I+ -oc -aa 0.150000 -ab 4 -ad 512 -ar 20 -as 32 -dj 0.700000 -dr 0 -ds 0.000000  -lr 8 -lw 0.050000  -sj 1.000000 -st 0.500000    -L 1000.000000  -N 148 -Dd  "c:\users\emanuel\appdata\local\temp\temp0\tmp\/static_system.dif.oct" < "c:\users\emanuel\appdata\local\temp\temp0\pts\points_0.pts" >> "c:\users\emanuel\appdata\local\temp\temp0\tmp\/static_system.dif.dc"
```

I have reinstalled DAYSIM multiple times without success.
Due to the reinstallation, if running `set` in the terminal it shows: 
```
RAYPATH=;C:\Daysim\lib\;.C:\Radiance\lib\;.;C:\Daysim\lib\;.C:\Radiance\lib\;.;C:\Daysim\lib\;.C:\Radiance\lib\;.;C:\Daysim\lib\;.C:\Radiance\lib\;.;C:\Daysim\lib\;.C:\Radiance\lib\;.;C:\Daysim\lib\;.C:\Radiance\lib\;.
```

Is this redundancy of the path causing the error? 

The setup:
- MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports)
- Parallels Desktop for Mac Business Edition 13.1.0 (43108)
- Windows 10 Pro
- Miniconda2 4.5.12 (64-bit)
- Python 2.7.15
- DAYSIM 3.1e (beta)

Help would be greatly appreciated!

**Comment by emanuel030** [comment]
Issue is resolved installing DAYSIM 4.0.

**Comment by daren-thomas** [comment]
@emanuel030 thank you for reporting back on resolving the issue! I have added an issue to mention the minimum Daysim version in the installation guide so future users don't run into this problem.
